meldebot/mel/gif.py

```diff
@@ -6,7 +6,10 @@
 # - MEL: Send random GIF (See `get_mel_params`). Usage: `/mel`
 # - MOTO: Send motorbike GIF. Usage: `/moto`
 ###############################################################################
-from telegram.ext import CommandHandler
+from typing import Callable, Dict, List, Union
+
+from telegram import Update
+from telegram.ext import CallbackContext, CommandHandler
```

Comment:
You can import TYPE_CHECKING and only import those classes when it's true (you will need to import annotations) as in:

```python
from __future__ import annotations
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from telegram import Update
```